src/components/Lobby.js

```diff
@@ -60,17 +60,21 @@ export default function Lobby() {
 
   // keyboard event handlers
   const moveUp = () => {
+    if (myPosY.current - delta < 0) return;
     myPosY.current -= delta;
 
     setMyPlayer(prevPlayer => {
       const newPlayer = JSON.parse(JSON.stringify(prevPlayer));
       newPlayer.y = myPosY.current;
       return newPlayer;
     });
+
+
   }
   useKey('ArrowUp', moveUp);
 
   const moveDown = () => {
+    if (myPosY.current + delta > 90) return;
```

Comment:
Yung 90 san galing to?

Reply:
@rsnakano

Reply:
arbitrary, pero based sa limited testing ko it works naman for a wide range of resolutions

Reply:
ay also max-width ng sprite ay 10 so 90 would be the best fit i think

Reply:
Mas okay siguro gumawa ng constant para one point of change lang if ever kailangan ayusin saka mas readable
```suggestion
    if (myPosY.current + delta > CONSTANTVARIABLE) return;
```

Reply:
@kquirapas

Reply:
ok ok